docs/webcast-install.md

```diff
@@ -0,0 +1,41 @@
+# How to apply V7 Webcast with Kaltura-live and Q&A plugins
+
+### Prerequisites
+If this is an on-prem and not served from Kaltura-SaaS you will need to make sure that: 
+- Core player version is `0.49.0-vamb.1` [installed](https://github.com/kaltura/playkit-js-contrib/releases/tag/kaltura-ovp-player%400.49.0-vamb.1) in your environment 
+- Q&A plugin is [installed](https://github.com/kaltura/playkit-js-qna/releases/tag/v0.7.5) in your environment
+- Kaltura-live plugin is [installed](https://github.com/kaltura/playkit-js-kaltura-live/releases/tag/v1.0.3) in your environment
+- Minimum MediaSpace version - 5.93.13
+
+### 1 Enable Webcast module in MediaSpace if missing
+Open your MediaSpace admin console, chose the relevant MediaSpace instance and click `manage` next to `Enable Modules`
+In the next screen - enable a module called `kwebcast` and hit `save`
+
+### 2 Activate the module in the MediaSpace instance
+Go into your MediaSpace instance admin (your MediaSpace url with the postfix `/admin`) and enable the kwebcast module
+Configure the different fields according to your needs 
+
+### 3 Create a V7 player and configure it
+In the KMC of the same account as the MediaSpace instance, go to the `Studio` tab and into `TV PLATFORM STUDIO` sub-tab.
```

Comment:
This is the flow I was aware of. But Yair want to check if we can have single uiconf to serve all beta partners instead of creating one for each of them. We need to get the approval of Gonen and if he accept to update the guide accordingly